security-admin/src/main/java/org/apache/ranger/rest/XUserREST.java

```diff
@@ -608,69 +622,6 @@ public VXLong countXGroupUsers(@Context HttpServletRequest request) {
 		return xUserMgr.getXGroupUserSearchCount(searchCriteria);
 	}
 
-	// Handle XGroupGroup
```

Comment:
Please see if the referenced methods (for REST APIs with path /groupgroups/.. ) in XUserMgr and XUserMgrBase can be removed as well?